### PR TITLE
[IMP] surveys:update get started docs

### PR DESCRIPTION
**Pull request by ksc-odoo**
Task ID: [2938139](https://www.odoo.com/web#id=2938139&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey There @mivu-odoo ,

New content and screenshots for the Surveys documentation. Can you please give this a review? Then I'll send it over to Zac.

Thanks!

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hello @ksc-odoo!

This will be part one of my final review; I figured I can publish these comments so you can get started on the changes while I continue reviewing the other RST files. 

This review only contains comments on the `create.rst` file. Please let me know if you have any questions about them. I will continue reviewing `scoring.rst` and `time_random.rst` in the meantime, and notify you when I finish making comments and publish them to this PR.

File: content/applications/marketing/surveys/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/surveys/overview/create.rst:31]: Gotcha. So, I deleted the original image, appropriately renamed the new screenshot your provided in the link above, and placed it in the proper create image folder. I did not, however, run the "git rm -imagepath..." command, as I figured this was replacing an image (with the same name), and not deleting it entirely. If you think that was done in error, please feel free to let me know, and I'll do my best to remedy the situation. Thanks!

File: content/applications/marketing/surveys/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/surveys/overview/create.rst:29]: Great catch. And, since the image its referring to was just replaced, I left this screenshot alone, and kept the updated version of the "questions-description-options.png" image.

File: content/applications/marketing/surveys/overview/create.rst
Comment (ksc-odoo) [reply content/applications/marketing/surveys/overview/create.rst:173]: To my knowledge, "Randomized per section" means that, when the survey is segmented into sections (i.e. "General Info" w/ 4 questions), EACH SECTION'S questions will be randomized. The order of sections remain the same, but the order of questions will be randomized, I believe.

Comment (mivu-odoo) [reply content/applications/marketing/surveys/overview/create.rst:173]: Got it! I would rewrite `...in other words, the number of random questions can be configured by section.` to better reflect that all the questions will be there, but the order of the questions will be randomized.

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [reply content/applications/marketing/surveys/overview/create.rst:31]: Hi @ksc-odoo!

So the original image would still need to be deleted using `git rm file-path` in order for Git to officially "recognize" the deletion. We can't right-click on the image file and click "Delete."

If you deleted the original image by right-clicking on the file and clicking "Delete," try to undo (CTRL + Z) until the original image is back in the folder. Then, follow the steps below:

1. Copy the original image file path.
2. On the terminal, run `git rm file-path`
3. Move the new image file into the folder.
4. On the terminal, run `git add .`

**Comment by mivu-odoo** [comment]
Hi @ksc-odoo!

I opened your feature branch up on my computer to see if I can tinker around with the build errors. Surprisingly, my terminal showed a different build error from the one you showed me earlier in the day. I'm wondering if the sphinx configuration on your computer might need to be updated, but that's a question for Zac when he comes back. 

I fixed the build errors that were showing on my terminal and pushed commit [1874eed](https://github.com/odoo/documentation/commit/1874eed9c5bc51ae07ef7a873b6fd8f420b803d5), and the PR is showing that all code checks have passed now.

After I make more comments on this PR, make sure to run `git pull --rebase` (Step 6 on my Discord message "Follow these steps to make another commit on an existing PR (squash commits) - Part 1") on your feature branch *before* you start making your next round of revisions, so my commit 1874eed is pulled onto your computer and you're working with the latest version of the branch.

**Comment by mivu-odoo** [comment]
PR was failing the ci/documentation code check - pushed commits [934e2b6](https://github.com/odoo/documentation/commit/934e2b67621fb2f1d0d766092c4cf04eaf3b17c5) and [36b7fc2](https://github.com/odoo/documentation/commit/36b7fc28b52915a97b290a470610dd2dbf1d0728) to delete trailing whitespaces.

PR is passing all checks now.

cc: @ksc-odoo @StraubCreative

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hello @ksc-odoo!

I left some comments on `scoring.rst`. There are slight differences in field names between Odoo 14 and Odoo 15. After this PR is merged and forward-ported, we can open another PR targeting the 15.0 branch.

I still have to review `time_random.rst`, and I might take another pass at `create.rst`. I will make another comment once I finish looking at those RST files. Feel free to start working on the changes to `scoring.rst` in the meantime. Thank you!

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:4]:
```suggestion

When creating a survey in Odoo, the user has the options to set a time limit on the survey and
randomize the questions.

```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:5]:
```suggestion
Time limit
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:9]:
```suggestion
a time limit on the survey ensures that every participant gets the same amount of time to fill in
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:10]:
```suggestion
answers. It also greatly reduces the chance of participants looking up responses via external
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:14]:
```suggestion
The :guilabel:` Survey Time Limit` setting is found in the :guilabel:`Options` tab of the survey
form under the :guilabel:`Questions` section.
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:16]: This screenshot should be retaken. I think the screenshot shows Odoo 15's interface, not Odoo 14, which is the target branch of this PR. Don't forget to compress your image file size as well with pngquant!

Before making changes to the RST:

1. Retake the screenshot (remember to zoom in 110-125% and make sure the final image width/length is less than 768px)
2. Compress the image file with pngquant
3. Save your compressed image in an accessible place

During making changes to RST (using the 'Follow these steps to make another commit on an existing PR (squash commits)' directions):

1. Do Steps 1-6 of 'Follow these steps to make another commit on an existing PR (squash commits)' directions
2. Copy the file path of the original image frontend-badges.png in the scoring image folder
3. Run `git rm [paste-file-path]` on your terminal
4. Cut the new, compressed image file and paste it into the scoring image folder
5. Continue from Step 7 of 'Follow these steps to make another commit on an existing PR (squash commits)' directions

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:22]:
```suggestion
participants keep track of the time remaining. Participants that do not submit their survey by the
preconfigured time limit will *not* have their answers saved.
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/time_random.rst:34]:
```suggestion
To randomize a survey, click the :guilabel:`Options` tab on the survey form. In the
:guilabel:`Questions` section, select :guilabel:`Randomized per section` for the
:guilabel:`Selection` field.
```

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Left some comments for `time_random.rst`! Last thing for me to do is to take a second look at `create.rst`, but feel free to work on the changes to `time_random.rst` and `scoring.rst` while I do that. Thank you!

**Comment by StraubCreative** [comment]
Updates as of [09b04e6](https://github.com/odoo/documentation/pull/2534/commits/09b04e682895c984a6f87860e241f6ff0e084ef0):

- solved merge conflicts across `scoring.rst` and `time_random.rst`.
- removed trailing whitespaces and fixed a couple line breaks in those docs as well.
- rebased on top of latest `14.0`

@ksc-odoo the CI checks are not passing on this latest commit due to bountiful trailing whitespaces. You can see them here: https://runbot.odoo.com/runbot/build/21751024.

The checks did not pass on prior commit [cc1af20](https://github.com/odoo/documentation/pull/2534/commits/cc1af207902b594957b491d6a59e4e59c2eca1cf) because there were merge conflicts against `14.0` base branch. This should have been identified and fixed before pushing that commit either by running `make html` or `git rebase 14.0`, in which case [either command would have communicated issues in the terminal readout](https://i.imgur.com/Dj0dzKl.png).

Reminders:

- if changes were made to your PR from other collaborators (e.g. solving a merge conflict, fixing syntax, etc.), be sure to run `git pull - rebase` command in your terminal so those changes are pulled to your computer before you continue making changes to the branch. Otherwise this creates duplicate work for maintainers (e.g. there should not be trailing whitespace issues [after MIVU fixed them for you](https://github.com/odoo/documentation/pull/2534#issuecomment-1341440193)).
- rebase on top of a fresh pull from your base branch _every time_ you're about to push commits (in this case, `14.0`).
- be mindful of trailing whitespaces and line breaks and do not ask for reviews until you are confident they have been addressed. PRs will not pass review if these are not 100% correct, and repeatedly ignoring these errors both risks introducing brittleness to your code and ultimately creates more work for everyone.
- do not ignore merge conflict warnings. If you need help solving them, just ask 🙂
- before pushing commits, run `make html` in your terminal to a) see a live preview of your changes in order to catch html rendering errors, and b) identify/address RST syntax errors that will be spit out in the terminal in red.

Tip: [this extension](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) should help out with trailing whitespaces, as it removes them whenever you save the file.

Before continuing content review with @mivu-odoo, please address all of the instructions in this comment and fix all of the CI errors until they pass.

Thank you and please reach out if you have any questions.

**Comment by StraubCreative** [comment]
<h3>Updates</h3> 

As of [09b04e6](https://github.com/odoo/documentation/pull/2534/commits/09b04e682895c984a6f87860e241f6ff0e084ef0):

- solved merge conflicts across `scoring.rst` and `time_random.rst`.
- removed trailing whitespaces and fixed a couple line breaks in those docs as well.
- rebased on top of latest `14.0`

-----

<h3>Notes</h3> 

@ksc-odoo the CI checks are not passing on this latest commit due to bountiful trailing whitespaces. You can see them here: https://runbot.odoo.com/runbot/build/21751024.

The checks did not pass on prior commit [cc1af20](https://github.com/odoo/documentation/pull/2534/commits/cc1af207902b594957b491d6a59e4e59c2eca1cf) because there were merge conflicts against `14.0` base branch. This would have been identified before pushing that commit either by running `make html` or `git rebase 14.0`, in which case [either command would have communicated issues in the terminal readout](https://i.imgur.com/Dj0dzKl.png).

<h4>Reminders</h4>

- if changes were made to your PR from other collaborators (e.g. solving a merge conflict, fixing syntax, etc.), be sure to run `git pull - rebase` command in your terminal so those changes are pulled to your computer before making more changes to the branch. Otherwise this creates duplicate work for maintainers (e.g. there should not be trailing whitespace issues [after MIVU already fixed them for you](https://github.com/odoo/documentation/pull/2534#issuecomment-1341440193)).
- rebase on top of a fresh pull from your base branch _every time_ you're about to push commits (in this case, `14.0`).
- be mindful of trailing whitespaces and line breaks and do not ask for reviews until you are confident they have been addressed. PRs will not pass review if these are not 100% correct, and repeatedly ignoring these errors both risks introducing brittleness to your code and ultimately creates more work for everyone.
- do not ignore merge conflict warnings. If you need help solving them, just ask 🙂
- before pushing commits, run `make html` in your terminal to a) see a live preview of your changes in order to catch html rendering errors, and b) identify/address RST syntax errors that will be spit out in the terminal in red.

--- 

<h3>Next steps</h3> 

**Before continuing content review with @mivu-odoo, please address all of the instructions in this comment and fix all of the CI errors until they pass. Reach out if you need help.**

Tip: [this extension](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces) should help out with trailing whitespaces, as it removes them whenever you save the file.

Thanks!

**Comment by StraubCreative** [comment]
@ksc-odoo checks are not passing on [8b82984](https://github.com/odoo/documentation/pull/2534/commits/8b82984b45000e8671e66c9a3a4453dd092e81f6).

The build errors a re listed on the [ci/documentation Details](https://runbot.odoo.com/runbot/build/21910413) link above.

Try again and only request a review when the checks pass.

Thank you.

**Comment by StraubCreative** [comment]
Hi @ksc-odoo 

I had to reset the branch to a previous commit due to the squashing error.

It appears the latest changes you made weren't actually committed (using `git add .` and `git commit -m` because  when I tried to reset to [8b82984](https://github.com/odoo/documentation/commit/8b82984b45000e8671e66c9a3a4453dd092e81f6) before the squash error occurred, I received an "unknown path" error.

This means I had to reset the branch further in the commit history than ideally we would have liked, to [09b04e6](https://github.com/odoo/documentation/commit/09b04e682895c984a6f87860e241f6ff0e084ef0). After rebasing 14.0 underneath it we now arrive at [44af0ab](https://github.com/odoo/documentation/commit/44af0ab042da574bbef08460e5ef811397e115da).

I'll be in touch with next steps.

**Comment by StraubCreative** [comment]
Checks pass as of [25a3499](https://github.com/odoo/documentation/pull/2534/commits/25a34998bfaec46b25e6867191f10d60f2ed408c).

Feel free to resume content review 💪 

Few things I noticed on the surface to take a look at @mivu-odoo :
- passive vs. active tense (e.g. "can be", "should", etc.)
- images: there are too many for `create.rst`. Most are either not helpful, improperly cropped, not in HD, or too big.
- content --> images: couple instances where written content relies too much on images. Let's break those dependencies wherever they show up.

When you're ready to do the next round of edits, go ahead and ping me @ksc-odoo and we'll do them together. We can review terminal commands, push/pull process, and all that other good stuff 😉

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:262]:
```suggestion
Once Odoo redirects the page to the survey template in the backend, make any necessary changes
```

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:37]:
```suggestion
Add questions and sections to the survey in the :guilabel:`Questions` tab. A section divides the
survey into parts; to make one, click :guilabel:`Add a section` and type in a section name. Then,
add questions or drag and drop questions into the divided sections.
```

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:69]:
```suggestion
   Different features appear in the :guilabel:`Answers` and :guilabel:`Options` tabs, depending on
   the :guilabel:`Question Type` chosen. However, the :guilabel:`Description` tab always remains the
   same, regardless of what question is chosen.
```

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:71]:
```suggestion
Create sections and questions
```

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:201]:
```suggestion
Manage access to the survey in the :guilabel:`Candidates` section.
```

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:42]: The width of this image needs to be cropped a little more. It's currently at a width of 975px, we need it to be 768px or less.

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:260]: This image can be deleted to get the image count down for this user doc.

File: content/applications/marketing/surveys/overview/create.rst
Comment (mivu-odoo) [inline content/applications/marketing/surveys/overview/create.rst:28]: This image looks fuzzy. Try retaking it please.

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hi @ksc-odoo!

I finished reviewing `create.rst` and left some comments. Please tag me again for another look when you push your revisions. Thank you 😸

**Comment by ksc-odoo** [comment]
Hey there @StraubCreative - I fully agree with the above suggested edits. Feel free to push any/all necessary changes. Thanks!

**Comment by StraubCreative** [comment]
Fixed commit history in [d9c55f5](https://github.com/odoo/documentation/pull/2534/commits/d9c55f5c28d2dc630245cb6498f664cfcb52f816).

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (StraubCreative) [inline content/applications/marketing/surveys/overview/time_random.rst:34]: Optional:
Would condense the two sentences together since they link to the same idea.
Suggested rewrite for second sentence for more context around use case.
```suggestion
time a participant begins the questionnaire. Using randomization as a survey method discourages
participants from looking at each other's responses, and helps control for individual testing.
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (StraubCreative) [inline content/applications/marketing/surveys/overview/time_random.rst:42]: Would combine these sentences together because they're a part of the same set of instructions (instead of splitting them into separate paragraphs). Slight rework to second section for flow.
```suggestion
:guilabel:`Selection` field. After enabling, navigate to the :guilabel:`Questions` tab and look in
the :guilabel:`Random questions count` column. From there, determine how many questions (per
section) Odoo should select and display during the shuffling of questions.
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

This was a pleasure to read. I like how the instructions were really broken down and easy to understand. I also appreciated how each doc was laid out with bite-size paragraphs.

I have a number of change requests to tighten everything up. There are a few CRs that must be addressed due to things like typo/grammar or line breaks, but a majority of them are more or less for your consideration as doc owner.

If you agree with the content changes listed below, I can push them up and we can pass over to DR right away. Otherwise let me know what you'd like to keep or disregard, per comment.

Thanks and looking forward to your response 🙂

**Comment by ksc-odoo** [comment]
> Hi @ksc-odoo
> 
> This was a pleasure to read. I like how the instructions were really broken down and easy to understand. I also appreciated how each doc was laid out with bite-size paragraphs.
> 
> I have a number of change requests to tighten everything up. There are a few CRs that must be addressed due to things like typo/grammar or line breaks, but a majority of them are more or less for your consideration as doc owner.
> 
> If you agree with the content changes listed below, I can push them up and we can pass over to DR right away. Otherwise let me know what you'd like to keep or disregard, per comment.
> 
> Thanks and looking forward to your response 🙂

@StraubCreative Feel free to make any changes you'd like to tighten things up. Do whatever you feel is best. Thanks

**Comment by StraubCreative** [comment]
Latest round of CRs addressed on [5c159be](https://github.com/odoo/documentation/pull/2534/commits/5c159be1c5074647a7a42cac95c18df5ded6ada4). 

These docs on Surveys are ready for your review @odoo/doc-review 😉

**Review by StraubCreative — APPROVED** [review]

File: content/applications/marketing/surveys/overview/create.rst
Comment (AntoineVDV) [inline content/applications/marketing/surveys/overview/create.rst:228]: Why? This list was [correctly indented](https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#be-consistent-with-indentation) before your change.

The same applies to the other lists on the page.

File: content/applications/marketing/surveys/overview/create.rst
Comment (AntoineVDV) [inline content/applications/marketing/surveys/overview/create.rst:144]: This doesn't need to be a list.
```suggestion
   :doc:`time_random`
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (AntoineVDV) [inline content/applications/marketing/surveys/overview/time_random.rst:23]: Borken role
```suggestion
When the :guilabel:`Survey Time Limit` option is checked, a timer will be displayed on every page
```

File: content/applications/marketing/surveys/overview/time_random.rst
Comment (AntoineVDV) [inline content/applications/marketing/surveys/overview/time_random.rst:16]: Broken role
```suggestion
Find the :guilabel:`Survey Time Limit` setting in the :guilabel:`Options` tab of the survey form,
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Feel free to merge after the comments have been resolved (approved or rejected).

@robodoo delegate+

**Comment by StraubCreative** [comment]
> I'm sorry, @StraubCreative. I'm afraid I can't do that.

@AntoineVDV robodoo being grumpy 😭

**Review by AntoineVDV — APPROVED** [review]
`delegate+` delegates the review rights to the author of the PR; `delegate=<someone>` delegates the rights to <someone>.

@robodoo r+

**Pull request closed without merging**
